src/ucs/debug/log.c

```diff
@@ -248,6 +249,53 @@ void ucs_log_print_compact(const char *str)
     }
 }
 
+void ucs_log_print_compact_multiline(const char *str)
```

Comment:
ucs_log_default_handler is already breaking the line by \n, but maybe it is not good enough because of the small buffer. can we improve ucs_log_default_handler to handle long multiline logs better instead of adding new function?

Reply:
oh, got it